Source/Diagnostics/ReducedDiags/ReducedDiags.H

```diff
@@ -42,6 +42,9 @@ public:
     /// separator in the output file
     std::string m_sep = " ";
 
+    /// precision for data in the output file
+    int m_precision = 14;
```

Comment:
```suggestion
    constexpr int default_precision = 14;
    int m_precision = default_precision;
```
I suggest this small change to improve readability.

Reply:
I can make this change, but note that none of the other variables have a separate constexpr to hold the default value.

Reply:
You are right. We should maybe address these issues all at once if at some point we decide to enforce the `readability-magic-numbers` check in `clang-tidy` !